_pytest/main.py

```diff
@@ -728,17 +730,57 @@ def _tryconvertpyarg(self, x):
         """Convert a dotted module name to path.
 
         """
-        import pkgutil
+        @contextlib.contextmanager
+        def _patched_find_module():
```

Comment:
This is probably better in the global namespace than a local function because it clutters the code quite a bit. Sorry I wasn't clear about that before.